tablecodec/tablecodec.go

```diff
@@ -244,11 +244,14 @@ func DecodeTableID(key kv.Key) int64 {
 
 // DecodeRowKey decodes the key and gets the handle.
 func DecodeRowKey(key kv.Key) (kv.Handle, error) {
-	if len(key) != RecordRowKeyLen || !hasTablePrefix(key) || !hasRecordPrefixSep(key[prefixLen-2:]) {
+	if len(key) < prefixLen || !hasTablePrefix(key) || !hasRecordPrefixSep(key[prefixLen-2:]) {
```

Comment:
The minimum common handle length is 9, so if len(key) < RecordRowKeyLen, we need to return error.

Reply:
Ok.